petastorm/tests/test_end_to_end_predicates_impl.py

```diff
@@ -37,3 +37,21 @@ def get_fields(self):
 
     def do_include(self, values):
         return self._values == values
+
+
+class VectorizedEqualPredicate(PredicateBase):
+    def __init__(self, values):
+        self._values = values
+
+    def get_fields(self):
+        return list(self._values.keys())
+
+    def do_include(self, values):
+        result = None
```

Comment:
It might be more concise/clear to write this as:
```
        result = [True] * len(values)
        for field_name in self._values.keys():
            result &= values[field_name] == self._values[field_name]
        return result
```

Reply:
Though actually that might just be my functional programming preference actually